config/locales/en.yml

```diff
@@ -899,7 +899,7 @@ en:
   bold_label_colon_value_html: "<strong>%{label}:</strong> %{value}"
   # Text inserted into a textarea when the bold button is clicked but no text was selected
   bold_text: bold text
-  bounding_box: Bounding Box
+  bounding_box: Clear boundary
```

Comment:
1. It's almost always better to change the translation key as well as the value, for the non-obvious reason that the way our translations are set up right now, translators will be notified about new keys, but not about changes to values of existing keys. Beyond technicalities, I think it just makes things easier to reason about when the keys match their values, e.g. when you're wondering what key defines the text "Clear boundary" you're probably going to look for `clear_boundary` first.
2. I'd like to stick with a noun for this text so that the part of speech is closer to the proper nouns of place names, so here that would be "Custom Boundary" I think rather than "Clear Boundary." I get that a verb makes more sense for a button, but I think this would be the least intrusive approach that's most consistent with the existing design.